src/rt/tracegc.d

```diff
@@ -40,14 +40,15 @@ version (none)
       }
         printf("\n");
       {
-        int[] a; delete a;
-        S[] as; delete as;
-        C c; delete c;
-        I i; delete i;
-        C* pc = &c; delete *pc;
-        I* pi = &i; delete *pi;
-        int* pint; delete pint;
-        S* ps; delete ps;
+        // delete is deprecated
+        //int[] a; delete a;
+        //S[] as; delete as;
+        //C c; delete c;
+        //I i; delete i;
+        //C* pc = &c; delete *pc;
+        //I* pi = &i; delete *pi;
+        //int* pint; delete pint;
+        //S* ps; delete ps;
```

Comment:
Note that this is in a `version(none)` block.

Reply:
Deprecation message issued at parse time ? :/

Reply:
`delete` has been deprecated for a year now.  It's time to turn it into an error.

Reply:
> Deprecation message issued at parse time ? :/

There are depreciation messages issued at lexing as well.

Reply:
Yup, but it looks that this has been fixed already:

https://issues.dlang.org/show_bug.cgi?id=18505